### DISCUSSION TITLE
Offers/Product/Invoices won't open - wrong URL?

### DISCUSSION THREAD

**Discussion by ironiemix** [baimard/gestion#145] [General]
Hello, 

my nextcloud is installed in a subdirectory:

https:/my.srv.tld/nextcloud

When I create a product, an offer or an invoice, I am unable to open the created object, because the link, that gets opened, when I click on the open button is

https:/my.srv.tld/nextcloud/nextcloud/index.php/apps/gestion/facture/1/show 
(does not work, created by button)

The subdir "nextcloud", where my nextcloud lives,  occurs two times in this URL. When i delete one occurance of "nextcloud" in this URL, the document opens just fine:

https:/my.srv.tld/nextcloud/index.php/apps/gestion/facture/1/show  
(works, double nextcloud deleted)

Am I missing a config setting here or is this a bug?

Frank

**Comment by baimard** [comment]
Dear Frank, 

What is your gestion version ?

https://github.com/baimard/gestion/issues/135

**Comment by ironiemix** [comment]
2.0.2

**Comment by ironiemix** [comment]
Oh, I see, there is 2.0.3 out now - will update in a moment. Sorry.

**Comment by ironiemix** [comment]
Hmm. I cannot install version 2.0.3, the Nextcloud App Store lists 2.0.2 as newest and im am too dump to install from tarfile, heres what I did:

sudo -u www-data php occ app:disable gestion
cd apps/
rm -rf gestion/
tar -xvzf ../gestion.tar.gz (downloaded from github)
chown -R www-data: gestion/
sudo -u www-data php occ app:enable gestion 
gives me : gestion 2.0.2 enabled (??? Why, I just downloaded version 2.0.3 from github?)
sudo -u www-data php occ app:update gestion

So still on 2.0.2, error persisting :(

**Comment by baimard** [comment]
Maybe I forget to change the version in the last release, but If you install this latest version in 2.0.3, you have the latest code.
